_data/accessibility-tests/footer.yml

```diff
@@ -0,0 +1,97 @@
+title: Footer
+component_status: pass
+test_items:
+# General tests
+  - summary: Some fields have auto-populate capability. 
+    summary_additional: When you navigate to an input field in the footer (like an email address), there will be an option in the browser to auto-populate information.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.5
+  - summary: Footer font meets color contrast requirements.
+    summary_additional: When you use a color contrast checker to view the footer, contrast between the font color and background color is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.3
+  - summary: Footer elements are visible and functional when screen is magnified.
+    summary_additional: When you zoom in to 400%, footer links and elements are stacked and visible. No left-to-right scrolling is required.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+  - summary: Footer icons and non-text content meet color contrast requirements. 
+    summary_additional: When you use a color contrast checker to view the footer, contrast between the icon or other non-text elements and adjacent colors is at least 3:1.
```

Comment:
Approved!

Reply:
@amycole501  @amyleadem @alex-hull 

**question:** Is it worth mentioning logo's here as well since there's a place for logo images? Or will "icon" satisfy the need? We mention "logos and other meaningful images" in another suggestion. 

I'm also curious how the buttons might fall into this category. After our discussion with pagination, I'm wondering if there's any concern with the icon button not meeting UI color contrast. Are these passable since the icons have the appropriate color, are focusable, and have the proper cursor pointer?

![image](https://github.com/user-attachments/assets/8c62a9ef-8604-437e-8d2a-08483fe7480d)
(In this example, the second button is hovered)

Reply:
It's an interesting question.

I think the word "icon" covers what we need to here, since the social media logos are technically implemented as icons. I worry that mentioning logos here will get into contrast checking on the agency logos themselves, which is important but probably beyond the scope of the footer accessibility. I think it is ok to leave the copy as is.

I'll defer to @amycole501 and @alex-hull about the icon background color.

Reply:
I agree with Amy. I think we can keep the "icon" and the other verbiage here. In terms of contrast, we do have a non-text contrast criteria (thinking that's the one you're mentioning, Charlie) which I think will help nudge individuals to check their color contrast of all non-text items. I also think our icons do meet the contrast standards as well.

Reply:
@alex-hull  answered in Slack that it should not be an issue due to the icons color being appropriate. Going to resolve this issue 👍

Reply:
I think it's fine to remove non-text, as I don't see anyone adding something other than logos and icons.